### PR TITLE
Adding IT cache and EGI organization

### LINKED ISSUE #192: Inconsistencies between osg and egi branches in lists of OSDF servers
In the egi branch I should be able to do `git diff osg domain.d/osgstorage.org.conf` and `git diff osg osgstorage-auth.conf` and see only difference lines beginning with `-` in the North America section of `subset of authenticated OSDF caches used by OSG`, and all the lines after `extra authenticated OSDF caches not used by OSG` should begin with `+`.   There are a few cases where that is not so.  Please make it consistent.

In addition, `git diff default domain.d/osgstorage.org.conf` should show no difference of servers between egi and default branches.

### PR DESCRIPTION
**Pull request by biozit**
Adding ds-914.cr.cnaf.infn.it cache and EGI organization

- Fixes #192

### CODE REVIEW

File: etc/cvmfs/domain.d/osgstorage.org.conf [hunk 1]
```diff
@@ -12,22 +12,30 @@ fi
...
+# North America
 CVMFS_EXTERNAL_URL="http://stashcache.t2.ucsd.edu:8000/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://dtn-pas.jack.nrp.internet2.edu:8000/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://mghpcc-cache.nationalresearchplatform.org:8000/"
 CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://osg-kansas-city-stashcache.nrp.internet2.edu:8000/"
 CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://osg-chicago-stashcache.nrp.internet2.edu:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://osg-new-york-stashcache.nrp.internet2.edu:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://daejeon-kreonet-net.nationalresearchplatform.org:8000/"
-CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;http://dtn-pas.jack.nrp.internet2.edu:8000/"
+CVMFS_EXTERNAL_URL="$CVMFS_EXTERNAL_URL;https//osg-new-york-stashcache.nrp.internet2.edu:8000/"
```
Comment (jthiltges) [inline etc/cvmfs/domain.d/osgstorage.org.conf:21]: From a quick glance, a couple of these are missing colons. Can you fix it?

**Review by DrDaveD — APPROVED** [review]

**Comment by DrDaveD** [comment]
Do `git diff master etc/cvmfs/domain.d/osgstorage.org.conf` to see what I'm seeing.

**Comment by biozit** [comment]
Screenshot 2024-02-06 at 3 05 56 PM

@DrDaveD Thank you for the pointers; please let me know if you want to change something.

**Review by DrDaveD — APPROVED** [review]

**Comment by DrDaveD** [comment]
@jose-caballero I leave up to you to merge and distribute.

**Comment by biozit** [comment]
@jose-caballero, could you let me know when this is ready?

**Comment by jose-caballero** [comment]
My apologies. The messages from GITHUB don't go to my work email. They go to a secondary address :) 
I haven't followed this thread. Let me read it.

**Pull request merged**